src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -971,50 +971,58 @@ LRESULT CALLBACK FancyZones::s_WndProc(HWND window, UINT message, WPARAM wparam,
 
 void FancyZones::UpdateZoneWindows() noexcept
 {
+    // Mapping between display device name and device index (operating system identifies each display device in the current session with an index value).

```

Comment:
let's remove the "current session" part, since it's just a speculation

Reply:
it's copy - paste from microsoft [documentation](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-enumdisplaydevicesw)

Reply:
this comment implies that the device idx is session dependent, and we don't know if that's the case...

Reply:
update